.github/workflows/publish.yml

```diff
@@ -60,3 +60,16 @@ jobs:
           helm-gcs-credentials: ${{ secrets.HELM_GCS_CREDENTIALS }}
           helm-gcs-repository: ${{ secrets.HELM_GCS_REPOSITORY }}
 
+  publish-release-notes:
+    needs: [ publish-artifacts, publish-helm-charts ]
+    runs-on: ubuntu-20.04
```

Comment:
Just curious, why only 20.04 version ?

Reply:
It's the most recent, and you always want a consistent CI environment (i.e. to pin a specific version) - https://github.com/actions/virtual-environments#available-environments